ninchatsdk/src/main/java/com/ninchat/sdk/activities/NinchatChatActivity.java

```diff
@@ -540,6 +540,7 @@ public void onConfigurationChanged(Configuration newConfig) {
         pip.setLayoutParams(pipLayoutParams);
 
         webRTCView.onResume();
+        messageAdapter.scrollToBottom(false);
```

Comment:
Isn't this a bit redundant since the `messageAdapter.scrollToBottom(false);` on line 581 is the one that gets called when keyboard pops up? This one gets called with all configuration changes.

Reply:
Thanks for pointing. Yes, it was not related to softphone fixes. I had to change it to catch orientation change and scroll to the bottom when orientation changed.